.github/actions/cascading/cascading.mjs

```diff
@@ -0,0 +1,80 @@
+import * as github from '@actions/github';
+import * as core from '@actions/core';
+import * as winston from 'winston';
+import {getOctokit} from '@actions/github';
+import {Cascading} from '@o3r/dev-tools';
+import {inspect, promisify} from 'util';
+import {exec} from 'child_process';
+
+const promisifiedExec = promisify(exec);
+
+const run = async () => {
+  try {
+    // Processing options from inputs
+    const noFf = core.getInput('noFf');
+    const cascadingPattern = core.getInput('cascadingPattern');
```

Comment:
Currently passed as a default option from the action.yml, isn't it enough ?
![image](https://user-images.githubusercontent.com/122815580/215075272-61bf7201-55bd-4e7f-9eab-21469082433d.png)

Reply:
nop, I am talking about `getInput('cascadingPattern', { required: true })`

Reply:
Since we provide a default value, won't it always be defined anyway ? I'd consider required : true only for parameters that are mandatory and without a default value.

Reply:
Let's keep it simple:
You expect a type `string | undefined`, you use `getInput('cascadingPattern')`.
You expect a type `string`, you use `getInput('cascadingPattern', { required: true })`.

According to `CascadingOptions`, you expect `string` then you are in the second case.
This is what ever is specified in you YAML file, especially that `inputs` can be provided via environment variables.